examples/elm/Main.elm

```diff
@@ -0,0 +1,121 @@
+module Main exposing (..)
+
+import Html exposing (Html, Attribute, h3, div, label, beginnerProgram, input, button, text, node)
+import Html.Attributes exposing (style, id, name, value, disabled, attribute, property)
+import Html.Events exposing (onClick, onInput, on)
+import Json.Encode as Encode
+import Json.Decode as Decode exposing (Decoder)
+
+
+main : Program Never Model Msg
+main =
+    beginnerProgram { model = model, update = update, view = view }
+
+
+type alias Model =
+    { disabled : Bool
+    , value : String
+    , name : String
+    , hasError : Bool
+    }
+
+
+type Msg
+    = ToggleDisabled
+    | ToggleValue
+    | ChangeValue String
+    | ChangeName String
+    | ToggleError
+
+
+model : Model
+model =
+    Model False "one" "form-field-name" False
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        ToggleDisabled ->
+            { model | disabled = not model.disabled }
+
+        ToggleValue ->
+            { model
+                | value =
+                    if model.value == "one" then
+                        "two"
+                    else
+                        "one"
+            }
+
+        ChangeValue val ->
+            let
+                _ =
+                    Debug.log "value changed from Elm" val
+            in
+                { model | value = val }
+
+        ChangeName name ->
+            { model | name = name }
+
+        ToggleError ->
+            { model | hasError = not model.hasError }
+
+
+onChange : (String -> Msg) -> Html.Attribute Msg
+onChange tagger =
+    on "onChange" (Decode.map tagger detailTargetValueDecoder)
+
+
+{-| IMPORTANT: We need to get the data of the original React event like
+   this : event.detail[0].target.value
+-}
+detailTargetValueDecoder : Decoder String
+detailTargetValueDecoder =
+    Decode.field "detail" <|
+        Decode.index 0 <|
+            Decode.at [ "target", "value" ] Decode.string
+
+
+{-| Create shorthand for custom element
+-}
+customSelect : List (Attribute msg) -> List (Html msg) -> Html msg
+customSelect =
+    node "custom-select"
+
+
+{-| Create shorthand for custom property. Property is easier to use
+than attribute for Boolean values.
+-}
+hasError : Bool -> Attribute Msg
+hasError =
+    property "hasError" << Encode.bool
+
+
+view : Model -> Html Msg
+view model =
+    div []
+        [ h3 [] [ text "Custom select component" ]
+        , div []
+            [ customSelect
+                [ id "customComponent"
+                , name model.name
+                , value model.value
```

Comment:
attributes `name` and `value` are passed JSON encoded (with escaped double quotes)
<img width="553" alt="screen shot 2017-06-23 at 23 43 12" src="https://user-images.githubusercontent.com/791764/27501494-1063ad66-586e-11e7-8385-630fcb837abb.png">


doesn't happen with: 
```elm
                , attribute "name" model.name
                , attribute "value" model.value
```